assets/js/modules/adsense/components/common/UseSnippetSwitch.js

```diff
@@ -86,11 +86,11 @@ export default function UseSnippetSwitch( props ) {
 					checked={ useSnippet }
 					disabled={ isDoingSaveUseSnippet }
 					hideLabel={ false }
-				/>{ ' ' }
+				/>
```

Comment:
We still need this whitespace (or we need to update css styles):
```suggestion
				/>{ ' ' }
```

Compare [this story](https://google.github.io/site-kit-wp/storybook/pull/7399/?path=/story/modules-adsense-components-setup-v2-setupmain--setup-account-site-ready-with-tag) with the one for the [develop branch](https://google.github.io/site-kit-wp/storybook/develop/?path=/story/modules-adsense-components-setup-v2-setupmain--setup-account-site-ready-with-tag). There must be a margin between the label and the badge.

Reply:
Ah, whoops, thanks for that. Added 👍🏻